src/data/faq.json

```diff
@@ -2239,19 +2477,29 @@
+                    "description": "The vaccination certificate is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has received a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a <a href='#glossary_proof_of_vaccination'>proof of vaccination</a>."
+                },
+                {
+                    "term": "Vaccination Certificate Token",
+                    "anchor": "vaccination_certificate_token",
+                    "description": "A QR code generated by vaccination centers or doctors' offices that can be scanned in the Corona alert app, for example, to document a vaccination."
+                },
+                {
+                    "term": "Vaccination Status",
+                    "anchor": "vaccination_status",
+                    "description": "Vaccination status includes information such as the number of vaccinations given, the vaccine used, and the validity of the vaccination."
                 },
                 {
                     "term": "Verifier-App / Checking a certificate / Verifying a certificate",
                     "anchor": "verify",
-                    "description": "The mere existence of a (digital) document has no verification/evidence power. Rather, <b>each document/certificate must</b> also be checked for validity and validation. <br/><br/> This check is done by a special app (in Germany the CovPassCheck app), the check app. <br/><br/> Individual countries may have different implementations of the check app. However all check apps use the same - consolidated on a European gateway - keys of the participating countries as well as the stored <a href='#glossary_business_rules'>Business Rules</a>. Therefore, all check apps will be able to validate all EU certificates in the future, as long as they follow the specification of the European eHealth Network."
+                    "description": "The mere existence of a (digital) document has no verification/evidence power. Rather, <b>each document/certificate must</b> also be checked for validity and validation. <br/><br/> This check is done by a special app (in Germany the <a href='#glossary_covpasscheck'>CovPassCheck app</a>), the check app. <br/><br/> Individual countries may have different implementations of the check app. However all check apps use the same - consolidated on a European gateway - keys of the participating countries as well as the stored <a href='#glossary_business_rules'>Business Rules</a>. Therefore, all check apps will be able to validate all EU certificates in the future, as long as they follow the specification of the European eHealth Network."

```

Comment:
"same - consolidated on a European gateway - keys"

change order to

"same keys - consolidated on a European gateway -"

otherwise it is just a literal translation of German grammar.

Reply:
Will be changed in the next commit, thanks!